blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpAggregatedIngestionHandler.java

```diff
@@ -99,10 +99,10 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
                         HttpResponseStatus.BAD_REQUEST );
             }
 
-        } catch (JsonParseException ex) {
+        } catch (JsonSyntaxException ex) {
             log.debug(String.format("BAD JSON: %s", body));
             log.error(ex.getMessage(), ex);
-            DefaultHandler.sendResponse(ctx, request, ex.getMessage(), HttpResponseStatus.BAD_REQUEST);
+            DefaultHandler.sendResponse(ctx, request, "Failed parsing Json content: " + ex.getMessage(), HttpResponseStatus.BAD_REQUEST);
```

Comment:
We have to be careful here that the exception message does not make the response an invalid Json. For example, if the exception message contains quotes, it will have to be escaped.